docs/01_introduction/README.md

```diff
@@ -117,18 +117,18 @@ which implements the ubiquitous ``configure``-``make``-``make install`` procedur
 A **software-specific** easyblock implements an installation procedure that is specific to a particular
 software packages. Infamous examples include the easyblocks we have for ``GCC``, ``OpenFOAM``, ``TensorFlow``, ...
 
-The installation procedure performed by an easyblock can be controlled by defining so-called
+The installation procedure performed by an easyblock can be locally customised by defining
```

Comment:
Hmm, easyconfig parameters are not really for customizations, that's what hooks (and plugins via `--include-*`) are for.

Maybe "can be steered" is better wording?